_includes/themes/zeppelin/_navigation.html

```diff
@@ -1,81 +1,98 @@
-                <li><a href="documentation.html">Older Versions</a></li>
-                <li role="separator" class="divider"></li>
-                <li><span><b>Snapshot</b>&nbsp;(development)<span></li>
-                <li><a href="/docs/0.8.0-SNAPSHOT">0.8.0-SNAPSHOT</a></li>
-              </ul>
-            </li>
+        <li><a href="/helium_packages.html">Helium</a></li>
 
-            <li><a href="/helium_packages.html">Helium</a></li>
+        <li class="docs">
+          <a href="#" data-toggle="dropdown" class="dropdown-toggle">Community<b class="caret"></b></a>
+          <ul class="dropdown-menu">
+            <li><a href="community.html">Contributors</a></li>
             <li><a href="https://github.com/apache/zeppelin">GitHub</a></li>
-
-            <!-- Apache -->
-            <li id="apache">
-              <a href="#" data-toggle="dropdown" class="dropdown-toggle">Apache<b class="caret"></b></a>
-               <ul class="dropdown-menu">
-                <li><a href="http://www.apache.org/foundation/how-it-works.html">Apache Software Foundation</a></li>
-                <li><a href="http://www.apache.org/licenses/">Apache License</a></li>
-                <li><a href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
-                <li><a href="/assets.html">Assets</a></li>
-                <li><a href="http://www.apache.org/foundation/thanks.html">Thanks</a></li>
-              </ul>
-            </li>
           </ul>
-        </nav><!--/.navbar-collapse -->
-      </div>
-    </div>
+        </li>
+          <!-- Apache -->
+        <li class="docs">
+          <a href="#" data-toggle="dropdown" class="dropdown-toggle">Apache<b class="caret"></b></a>
+          <ul class="dropdown-menu">
+            <li><a href="http://www.apache.org/foundation/how-it-works.html">Apache Software Foundation</a></li>
+            <li><a href="http://www.apache.org/licenses/">Apache License</a></li>
+            <li><a href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
+            <li><a href="/assets.html">Assets</a></li>
+            <li><a href="http://www.apache.org/foundation/thanks.html">Thanks</a></li>
+          </ul>
+        </li>
+      </ul>
+    </nav><!--/.navbar-collapse -->
+  </div>
+</div>
 
 {% if page.title == "Zeppelin" %}
 <div class="jumbotron">
-  <div class="container">
-    <h1>Apache Zeppelin</h1>
-    <p>A web-based notebook that enables interactive data analytics. <br/>
-      You can make beautiful data-driven, interactive and collaborative documents with SQL, Scala and more.
-    </p>
+  <div class="container" style="padding-bottom: 5px;">
+    <div>
+      <div class="zeppelin">
+        <div class="zeppelin2"></div>
+      </div>
+    </div>
+    <div style="margin-top: -250px; position: relative;">
+      <h1 style="font-size:40px;">Apache Zeppelin</h1>
+      <p class="description">
+        A web-based notebook that enables interactive data analytics for
```

Comment:
1. Text for Section 1

A. Remove **interactive** since it's duplicated with `interactive and ...`
B. Or use the original text

> A web-based notebook that enables interactive data analytics. 
You can make beautiful data-driven, interactive and collaborative documents with SQL, Scala and more.

But +1 for **A** because it's simpler and short. IMO, we don't need some words like `you` and `make` in the front page title.